packages/editor/src/lib/TldrawEditor.tsx

```diff
@@ -516,3 +524,118 @@ export function useOnMount(onMount?: TLOnMountHandler) {
 		if (editor) return onMountEvent?.(editor)
 	}, [editor, onMountEvent])
 }
+
+const WATERMARK_SRC = `url('data:image/svg+xml;utf8,${encodeURIComponent(watermarkDesktopSvg)}') center 100% / 100% no-repeat`
+
+const Watermark = React.memo(() => {
+	const events = useCanvasEvents()
+
+	const ref = useRef<HTMLAnchorElement>(null)
+	useLayoutEffect(() => {
+		// eslint-disable-next-line deprecation/deprecation
+		if (ref?.current) ref.current.style.webkitMask = WATERMARK_SRC
+	}, [])
+
+	const editor = useEditor()
+	const showWatermark = useValue(
+		'show watermark',
+		() => {
+			const { width } = editor.getViewportScreenBounds()
+			if (width < 760) return false
+			if (!featureFlags.enableLicensing.get()) return false
+			return editor.getLicenseState() === 'unlicensed'
```

Comment:
using strings like this is a bit brittle. let's use const, enum-like set of strings so this doesn't get messed up and can be type-checked better

Reply:
Good call